rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -256,7 +256,7 @@ The protocol generally operates as follows:
     * Highest Lamport Clock value over all transactions (LC).
     
 2. When receiving Bob's message, Alice compares the XOR value from Bob with its own (§6.2.2):
-    * When the XOR is the same and the LC value equals Alice's highest Lamport Clock value: no action required.
```

Comment:
This condition had a reason. Maybe @woutslakhorst can remember what that was?
(its not in the code (anymore))

Reply:
Note that it is not implemented

Reply:
```// nothing to do if peers are now synced
	if xor.Equals(hash.FromSlice(msg.XOR)) {
		return nil
	}```
in `handlers.go#handleState`?

Reply:
Might be a leftover when XOR was not over all transactions...

Reply:
Makes sense. And would be fine to drop the requirement in that case